examples/llava/clip.cpp

```diff
@@ -2074,5 +2081,6 @@ int clip_n_mmproj_embd(const struct clip_ctx * ctx) {
     }
 
     std::string proj_type = PROJECTOR_TYPE_NAMES[ctx->proj_type];
-    throw std::runtime_error(format("%s: don't support projector with: %s currently\n", __func__, proj_type.c_str()));
+    LOG_TEE("%s: don't support projector with: %s currently\n", __func__, proj_type.c_str());
+    return 0;
```

Comment:
This should never happen since the projector type is already checked when loading the model, right? If that's the case, it would be better to use an assert here, since returning zero would allow applications to continue without catching the error, and this should never happen unless there is a bug somewhere else anyway.